library/Versions.re

```diff
@@ -217,10 +225,10 @@ let getRemoteVersions = () => {
   let%lwt installedVersions = Remote.getInstalledVersionSet();
 
   versions
-  |> Core.List.filter(~f=x =>
+  |> Base.List.filter(~f=x =>
```

Comment:
I have zero idea why I have used `Core.List` for simply mapping and filtering, but what do you think if we're still here dropping this? `Base.List.map |> List.sort` feels weird